client/src/main/java/com/aapeli/colorgui/ColorButton.java

```diff
@@ -401,12 +400,9 @@ public void processActionEvent() {
         synchronized (this.aVector3280) {
             if (this.aVector3280.size() != 0) {
                 ActionEvent var2 = new ActionEvent(this, 1001, this.aString3272);
-                Enumeration<ActionListener> var3 = this.aVector3280.elements();
-
-                while (var3.hasMoreElements()) {
-                    var3.nextElement().actionPerformed(var2);
+                for (ActionListener listener : aVector3280) {
```

Comment:
Could you deobfuscate `aVector3280` to `listeners` or something else?

Reply:
I have a local branch where I convert the codebase from using `Vector`s everywhere to `List`s and that's exactly what I do there: https://github.com/StenAL/playforia-minigolf/commit/7befbe59716fa179a3de580f9e159a8b5a0cd44b

I'd prefer to do the deobfuscation as part of the follow-up PR with that commit instead of backporting the changes here if that works for you.

Reply:
Okay, fine :)